lib/bolt/transport/local/shell.rb

```diff
@@ -184,7 +184,7 @@ def execute(command, sudoable: true, **options)
 
             begin
               if writable && index < in_buffer.length
```

Comment:
Do we need to also change `length` here (and below) to `bytesize`?

Perhaps we ought to make a copy of `in_buffer` encoded as binary and then the existing algorithm should work.